syntax = "proto3";

service ScoringPoseService {
  rpc loadService (loadData) returns (loadStatus) {}
  rpc getScore (ImageRequest) returns (ScoreResponse) {}
}

message loadData {}

message loadStatus {
  string status = 1;
}

message Image {
  string filename = 1;
  string format = 2;
  int32 width = 3;
  int32 height = 4;
  bytes data = 5;
}

message ImageRequest {
  Image image = 1;
  int32 time = 2;
}

message ScoreResponse {
  double score = 1;
  int32 time = 2;
}